refarch-integrations/refarch-address-integration/refarch-address-integration-client/src/test/java/de/muenchen/refarch/integration/address/client/impl/AddressGermanyImplTest.java

```diff
@@ -0,0 +1,132 @@
+package de.muenchen.refarch.integration.address.client.impl;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.when;
+
+import de.muenchen.refarch.integration.address.client.api.AddressGermanyApi;
+import de.muenchen.refarch.integration.address.client.exception.AddressServiceIntegrationClientErrorException;
+import de.muenchen.refarch.integration.address.client.exception.AddressServiceIntegrationException;
+import de.muenchen.refarch.integration.address.client.exception.AddressServiceIntegrationServerErrorException;
+import de.muenchen.refarch.integration.address.client.model.request.SearchAddressesGermanyModel;
+import de.muenchen.refarch.integration.address.client.gen.api.AdressenBundesweitApi;
+import de.muenchen.refarch.integration.address.client.gen.model.AddressServicePage;
+import de.muenchen.refarch.integration.address.client.gen.model.BundesweiteAdresse;
+import de.muenchen.refarch.integration.address.client.gen.model.BundesweiteAdresseResponse;
+import de.muenchen.refarch.integration.address.client.gen.model.BundesweiteAdresseResponseItem;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.HttpServerErrorException;
+import org.springframework.web.client.RestClientException;
+import reactor.core.publisher.Mono;
+
+class AddressGermanyImplTest {
+
+    public static final String QUERY = "Sample Query";
+    public static final String ZIP = "12345";
+    public static final String CITY = "Sample City";
+    public static final String GEMEINDESCHLUESSEL = "456789";
+    public static final String SORT = "name";
+    public static final String SORTDIR = "asc";
+    private final AdressenBundesweitApi adressenBundesweitApi = Mockito.mock(AdressenBundesweitApi.class);
```

Comment:
You should import `mock` statically to aligned with the other tests in other integrations. Might apply to other tests in this integration as well. Suggest to search globall for `Mockito.mock`

Reply:
Done